src/renderers/dom/fiber/ReactDOMFiberEntry.js

```diff
@@ -499,8 +499,22 @@ function renderSubtreeIntoContainer(
     // First clear any existing content.
     // TODO: Figure out the best heuristic here.
     if (!shouldReuseContent(container)) {
+      let warned = false;
       while (container.lastChild) {
-        container.removeChild(container.lastChild);
+        let rootSibling  = container.lastChild;
```

Comment:
Nit: we read it twice here, can we avoid that? 
Supernit: I see two spaces before `=`